numpy/random/mtrand.pyi

```diff
@@ -0,0 +1,132 @@
+import builtins
+from typing import overload, Optional, Sequence, Tuple, TypeVar, Union
+
+from numpy import ndarray
+
+_Size = Union[int, Tuple[int, ...]]
+_T = TypeVar('_T')
+
+class RandomState:
+    def __init__(self, state: Optional[Union[int, ndarray]] = ...) -> None: ...
```

Comment:
The constructor argument is called `seed`, not `state`:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.RandomState.html

Also, it can accept sequences of integers ("array_like") not just ndarray instances.